### ISSUE TITLE
Viewing comments

### ISSUE THREAD

**Issue by stemy2** [cloudrac3r/bibliogram#25]
Hello,

You said you will not add the feature "comment". Does it means that we will never be able to comment or be able to see comments ?

**Comment by cloudrac3r** [comment]
Creating comments, like any other content, will not be allowed because that is out of scope of this project.

While viewing other people's comments _is_ in the scope of this project, I consider them to be mainly a waste of screen space and reader time. Unlike Twitter, where replies are an integral part of the platform, I believe that Instagram would be better if comments did not exist. I do not want to waste my time coding additional visual garbage to display on Bibliogram, especially when there are much better things I could be doing instead.

But that's just my opinion. If you disagree with my opinion, please let me know.

There is a much higher chance of me changing my mind about this after I actually add a user settings page, so that individual users can opt out of features like this.

**Comment by TheFrenchGhosty** [comment]
>Unlike Twitter, where replies are an integral part of the platform, I believe that Instagram would be better if comments did not exist. I do not want to waste my time coding additional visual garbage to display on Bibliogram, especially when there are much better things I could be doing instead.

I agree, Twitter replies are important because they are part of the content, Instagram is a pictures sharing site, I don't care about the opinion of others about those pictures

**Comment by cloudrac3r** [comment]
Waiting on #28 before I will think about this again.

**Comment by mehkera** [comment]
I would be nice to make the comments hidden by default !
Indeed , comments are garbage in instagram due to the fact that people comments to get well and have subscribers ! and comments are always not relevant at all !!!!

**Comment by zoj613** [comment]
making viewing comments optional would be really nice. I read comments for entertainment purposes on some posts.

**Comment by mendel5** [comment]
+1 from me for an option to read comments (see also https://github.com/cloudrac3r/bibliogram/issues/45 )

Reading comments makes using Instagram/ Bibliogram more multi-dimensional :)

**Comment by cloudrac3r** [comment]
The settings page exists now and I have a template for the control so yeah I'll get to this at some point.

**Comment by cloudrac3r** [comment]
Bibliogram is moving to sourcehut, so this GitHub thread should not be commented on in future. I won't actually lock the thread out of courtesy.

If anyone has further comments, please post them here instead: https://todo.sr.ht/~cadence/bibliogram-issues/7
